### ISSUE TITLE
Are there Instructions? 

### ISSUE THREAD

**Issue by ghost** [wpatter6/clickr#7]
I can’t figure out how to move the mouse or cursor and what the middle button does.

**Comment by wpatter6** [comment]
Yeah I need to put together some instructions to show in the app...

Basically on the big circle, you tap the left or right arrows to tab left and right, while the up and down arrows move "sections" up and down (depending on the accessibility of the site).  

The middle button behaves as a click.  Currently there is no mouse behavior but something might come eventually.

Video controls and bookmarks should be more straightforward
